docs/docs/index.md

```diff
@@ -144,7 +144,9 @@ gives you back
 
 !!! info "Find more examples [here](resources/examples.md)."
 
-As you can see, dataview doesn't only allow you to aggregate your data swiftly and always up to date, it also can help you with operations to give you new insights on your dataset. Browse through the documentation to find out more on how to interact with your data - have fun exploring your vault! 
+As you can see, dataview doesn't only allow you to aggregate your data swiftly and always up to date, it also can help you with operations to give you new insights on your dataset. Browse through the documentation to find out more on how to interact with your data.
```

Comment:
Something feels off about the phrasing here. I'll revisit this.

Reply:
Attempt 1: Dataview allows you to swiftly query an always up-to-date index of your metadata. [...]

Sounds like technical nonsense to me.

Reply:
Attempt 2: Dataview lets you quickly display queries against your vault metadata in real-time. [...]

Sounds sensible but still not ideal IMO.

Reply:
Can't think of anything else at the moment.